table_sync.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rubocop-config-umbrellio", "~> 0.70"
   spec.add_development_dependency "simplecov", "~> 0.16"
 
-  spec.add_development_dependency "activejob"
+  spec.add_development_dependency "activejob", ">= 4.2.11"
```

Comment:
- you should provide customized `gemfile` list for CI with all supported versions (all versions since `4.2`) (but this approach requires some `.travis-ci.yml` updates)
- i think we should require `>= 4.2`

Reply:
What do you think  about testing only major updates, e.g 4.2 and 5.2?

Reply:
I don't think we want to keep Rails 4 support for a long period of time so let's push it as is?